### DISCUSSION TITLE
Any progress on supabase-workflows

### DISCUSSION THREAD

**Discussion by razakpm** [supabase/supabase#14391] [Questions]
I was looking [Supabase workflows](https://supabase.com/blog/supabase-workflows) and Event-queue concept for my project. Didn't see any update after April 2021, do we have any other alternative?

**Comment by vickkhera** [comment]
I've found a couple of good contenders, depending how you like to work. [Windmill](https://www.windmill.dev) is GUI focused, and [Inngest](https://www.inngest.com) is code focused. I prefer the latter development style.

Windmill runs everything in their own infrastructure and workers. Inngest runs the coordinator and calls functions via API on your own workers. They don't currently offer a Supabase functions worker option so I'm running my workers inside my Next.js app which works just splendidly.
